server/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable react/react-in-jsx-scope */
+/* eslint-disable react/jsx-filename-extension */
 import path from 'path';
 
 import dotenv from 'dotenv';
```

Comment:
Let's checkout the CLI for dotenv. We want to avoid importing dev dependencies into our app.